### PR TITLE
Updated Release Notes for veteran PACT status

### PR DESCRIPTION
**Pull request by jjennings-va**

### CODE REVIEW

File: src/content/apiDocs/health/communityCareReleaseNotes.mdx [hunk 1]
```diff
@@ -1,3 +1,10 @@
+### July 23, 2021
+CCE now presents and accounts for a veterans PACT status (Patient Aligned Care Team) when determining eligibility
```
Comment (as-va) [inline src/content/apiDocs/health/communityCareReleaseNotes.mdx:2]: veterans -> veteran's

Comment (as-va) [reply src/content/apiDocs/health/communityCareReleaseNotes.mdx:2]: `CCE` should probably be `the CCE API` or simply `the API`.

Comment (as-va) [reply src/content/apiDocs/health/communityCareReleaseNotes.mdx:2]: (Next sentences): "PACT status is only consulted for `PrimaryCare` requests. A veteran with an active PACT is ineligible for community care for Primary Care appointments.

Additionally, the new field `pactStatus` has been added to the API response when it has been used in the `eligibility` determination."

**Review by as-va — CHANGES_REQUESTED** [review]

File: src/content/apiDocs/health/communityCareReleaseNotes.mdx (hunk 1 above)
Comment (jjennings-va) [reply src/content/apiDocs/health/communityCareReleaseNotes.mdx]: Made these changes, thanks!

**Review by as-va — DISMISSED** [review]

**Review by gileswells — APPROVED** [review]
LGTM

**Pull request merged**